worker/push/push.go

```diff
@@ -104,17 +104,19 @@ func Worker(ctx *job.WorkerContext) error {
 	if err != nil {
 		return err
 	}
+	if len(cs) > 10 {
+		ctx.Logger().Warnf("too many notifiable devices: %d", len(cs))
+		cs = cs[:10]
+	}
```

Comment:
Why adding this arbitrary limit ?

Reply:
I think we don't want to send so many pushes for the same notification

Reply:
Maybe we should prioritize on the latest devices, I think here the oldest devices will be used.

The logging string could also be a bit clearer that notifiables are getting sliced, "too many notifiable devices %s, capping to %s", len(cs), MAX_NOTIFIABLE_DEVICES".

Instead of a magic number, a CONSTANT would be clearer.